rush-plugins/rush-serve-plugin/src/phasedCommandHandler.ts

```diff
@@ -153,12 +157,33 @@ export async function phasedCommandHandler(options: IPhasedCommandHandlerOptions
 
       const serveConfig: RushServeConfiguration = new RushServeConfiguration();
 
-      const routingRules: Iterable<IRoutingRule> = await serveConfig.loadProjectConfigsAsync(
+      const routingRules: IRoutingRule[] = await serveConfig.loadProjectConfigsAsync(
         selectedProjects,
         logger.terminal,
         globalRoutingRules
       );
 
+      const { logServePath } = options;
+      if (logServePath) {
+        for (const project of selectedProjects) {
+          const projectLogServePath: string = `${logServePath}/${project.packageName}`;
```

Comment:
Make this into a common function that you can consume in convertToLogFileUrls as well. Seems like the logic of constructing this path should be common